src/api/app/models/user.rb

```diff
@@ -752,7 +752,7 @@ def involved_patchinfos
   def user_relevant_packages_for_status
     role_id = Role.hashed['maintainer'].id
     # First fetch the project ids
-    projects_ids = involved_projects.pluck(:id)
+    projects_ids = involved_projects.select(:id)
     packages = Package.joins("LEFT OUTER JOIN relationships ON (relationships.package_id = packages.id AND relationships.role_id = #{role_id})")
     # No maintainers
     packages = packages.where([
```

Comment:
as you're touching the function anyway, I would move the final pluck to the calling function (there is only one). As this function is user_relevant_packages_for_status (for whatever the user_ prefix is good for in User model :) and not suggesting it's returning ids.

Reply:
I agree, but that has nothing to  do with efficiency. :confused:

Reply:
I would agree with @Ana06, but I know to many occasions where you demand similar additions from others. I am torn:laughing:

Reply:
`Demand != suggest`

I find great @coolo's suggestion but I will implement it in a different PR. It is out of the scope.